packages/notifi-react/lib/context/NotifiTargetContext.tsx

```diff
@@ -661,6 +668,151 @@ export const NotifiTargetContextProvider: FC<PropsWithChildren> = ({
     [],
   );
 
+  const getSignature = useCallback(
+    async (message: Uint8Array | string) => {
+      let signature: Uint8Array | string = '';
+
+      // TODO: Add logic for rest of the chains
+      switch (walletWithSignParams.walletBlockchain) {
+        case 'AVALANCHE':
+        case 'ETHEREUM':
+          signature = await walletWithSignParams.signMessage(
+            message as Uint8Array,
+          );
+          break;
+        default:
+          signature = '';
```

Comment:
Shall we handle an error here ex. setError('ERROR: not supported chain') just in case.